src/ClientInfo.h

```diff
@@ -78,6 +78,8 @@ class ClientInfo : public hash_link
     unsigned int quotaPeekReserv() const; ///< returns the next reserv. to pop
     void quotaDequeue(); ///< pops queue head from queue
     void kickQuotaQueue(); ///< schedule commHandleWriteHelper call
+    /// selects the delayed head descriptor for writing or quotaDequeue()
```

Comment:
Let's polish to avoid ambiguous "or" binding:

```suggestion
    /// either selects the head descriptor for writing or calls quotaDequeue()
```

Reply:
Done.